nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -1207,14 +1215,6 @@ def forward(
             # Post-LN: x -> MHA -> Residual -> LN -> MLP -> Residual -> LN
             # Normformer: x -> LN -> MHA -> LN -> Residual -> MLP (w/LN) -> Residual
 
-            if type(hidden_states) is tuple:
-                if len(hidden_states) == 2:
-                    hidden_states, position_bias = hidden_states
-                elif len(hidden_states) == 3:
-                    hidden_states, position_bias, encoder_decoder_position_bias = hidden_states
-                else:
-                    raise IndexError('Hidden_states needs to be tuple containing 2 or 3 elements.')
-
```

Comment:
Can you delete this?

Reply:
I think he moved it further down

if type(x_) is tuple:
                        if len(x_) == 2:
                            x_, position_bias = x_
                        elif len(x_) == 3:
                            x_, position_bias, encoder_decoder_position_bias = x_
                        else:
                            raise IndexError('Hidden_states (x_) needs to be tuple containing 2 or 3 elements.')